WiserTaskScheduler/WiserTaskScheduler/Core/Services/ParentUpdateService.cs

```diff
@@ -124,6 +137,30 @@ private async Task ParentsUpdateMainAsync(IDatabaseConnection databaseConnection
             }
         }
         
+        /// <summary>
+        /// The optimize routine, performs the optimize query on the targeted databse
+        /// </summary>
+        /// <param name="databaseConnection">The database connection to use.</param>
+        /// <param name="databaseHelpersService">The <see cref="IDatabaseHelpersService"/> to use.</param>
+        /// <param name="targetDatabase">The database we are applying the parent updates on.</param>
+        private async Task ParentsUpdateOptimizeTables(IDatabaseConnection databaseConnection, IDatabaseHelpersService databaseHelpersService, ParentUpdateDatabaseStrings targetDatabase)
+        {
+            if (await databaseHelpersService.DatabaseExistsAsync(targetDatabase.DatabaseName))
+            {
+                if (await databaseHelpersService.TableExistsAsync(WiserTableNames.WiserParentUpdates, targetDatabase.DatabaseName))
+                {
+                    try
+                    {
+                        await databaseConnection.ExecuteAsync(targetDatabase.OptimizeQuery);
+                    }
+                    catch (Exception e)
+                    {
+                        await logService.LogError(logger, LogScopes.RunBody, LogSettings, $"Failed to run query ( {targetDatabase.OptimizeQuery} ) in parent update service due to exception:{Environment.NewLine}{Environment.NewLine}{e}", "ParentUpdateService");
+                    }
+                }
+            }
```

Comment:
I don't deny the PR for this, but next time please don't indent so much. Instead of that, make it like this, this makes it easier to read and maintain:
```C#
private async Task ParentsUpdateOptimizeTables(IDatabaseConnection databaseConnection, IDatabaseHelpersService databaseHelpersService, ParentUpdateDatabaseStrings targetDatabase)
{
    if (!await databaseHelpersService.DatabaseExistsAsync(targetDatabase.DatabaseName))
    {
        return;
    }
    
    if (!await databaseHelpersService.TableExistsAsync(WiserTableNames.WiserParentUpdates, targetDatabase.DatabaseName))
    {
        return;
    }

    try
    {
        await databaseConnection.ExecuteAsync(targetDatabase.OptimizeQuery);
    }
    catch (Exception e)
    {
        await logService.LogError(logger, LogScopes.RunBody, LogSettings, $"Failed to run query ( {targetDatabase.OptimizeQuery} ) in parent update service due to exception:{Environment.NewLine}{Environment.NewLine}{e}", "ParentUpdateService");
    }
}
```

Or even better, like this:
```C#
private async Task ParentsUpdateOptimizeTables(IDatabaseConnection databaseConnection, IDatabaseHelpersService databaseHelpersService, ParentUpdateDatabaseStrings targetDatabase)
{
    if (!await databaseHelpersService.DatabaseExistsAsync(targetDatabase.DatabaseName) || !await databaseHelpersService.TableExistsAsync(WiserTableNames.WiserParentUpdates, targetDatabase.DatabaseName))
    {
        return;
    }

    try
    {
        await databaseConnection.ExecuteAsync(targetDatabase.OptimizeQuery);
    }
    catch (Exception e)
    {
        await logService.LogError(logger, LogScopes.RunBody, LogSettings, $"Failed to run query ( {targetDatabase.OptimizeQuery} ) in parent update service due to exception:{Environment.NewLine}{Environment.NewLine}{e}", "ParentUpdateService");
    }
}
```